server/athenian/api/models/state/models.py

```diff
@@ -640,3 +640,19 @@ class DashboardChartGroupBy(create_time_mixin(created_at=True, updated_at=True),
     DashboardChartGroupBy.jira_issue_types,
     DashboardChartGroupBy.jira_labels,
 )
+
+
+class InstallationProgress(create_time_mixin(created_at=False, updated_at=False), Base):
+    """Tracking installation progress for github accounts"""
+
+    __tablename__ = "installation_progress"
+
+    github_account_id = Column(Integer(), primary_key=True)
+    account_created = Column(TIMESTAMP(timezone=True))
+    fetch_started = Column(TIMESTAMP(timezone=True))
+    fetch_completed = Column(TIMESTAMP(timezone=True))
+    consistency_started = Column(TIMESTAMP(timezone=True))
+    consistency_completed = Column(TIMESTAMP(timezone=True))
+    precompute_started = Column(TIMESTAMP(timezone=True))
+    precompute_completed = Column(TIMESTAMP(timezone=True))
```

Comment:
This is  reposet and Athenian account specific. We have to move those timestamps to the `repository_sets` table.

Reply:
so this force us to have some key in the state DB for github_account_ids.
create a table for comparing github_account_id and athenian_account_id needed.

Reply:
You probably mean `account_github_accounts`?

Reply:
if it is contains both this IDs - yes it is

Reply:
Then yes it is